### DISCUSSION TITLE
설치 후 최초 로그인 시 화면 내 안내가 있으면 좋을 것 같습니다.

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#4] [제보하기]
최초 설치 후 gmail 인증을 통해서 로그인하면, 스페이스가 선택되지 않은 상태입니다.

Space에 Join 되기 전에 랜딩 페이지 같이 안내가 필요할 것 같습니다. 
캐릭터도 안 나오고 에러 화면처럼 보였어요.

**Comment by shlee303** [comment]
사용자에게 안내 될 수 있도록 Beta 전에는 추가 하면 좋을 것 같네요! 좋은 의견 감사합니다.
